src/mockServiceWorker.js

```diff
@@ -68,6 +68,7 @@ self.addEventListener('message', async function (event) {
             frameType: client.frameType,
           },
         },
+        clientId,
```

Comment:
Why forward `clientId` with this event if the library doesn't process `clientId` passed with this event? Does this have some hidden effect you noticed?